jans-cedarling/cedarling/src/jwt/test/with_validation/key_service.rs

```diff
@@ -245,9 +249,12 @@ fn can_update_local_jwks() {
     });
 
     // generate the signed token strings
-    let access_token = generate_token_using_claims(&access_token_claims, &encoding_keys[0]);
-    let id_token = generate_token_using_claims(&id_token_claims, &encoding_keys[1]);
-    let userinfo_token = generate_token_using_claims(&userinfo_token_claims, &encoding_keys[0]);
+    let access_token = generate_token_using_claims(&access_token_claims, &encoding_keys[0])
+        .unwrap_or_else(|e| panic!("Failed to generate access_token: {:?}", e));
+    let id_token = generate_token_using_claims(&id_token_claims, &encoding_keys[1])
+        .unwrap_or_else(|e| panic!("Failed to generate id_token: {:?}", e));
+    let userinfo_token = generate_token_using_claims(&userinfo_token_claims, &encoding_keys[0])
+        .unwrap_or_else(|e| panic!("Failed to generate userinfo_token: {:?}", e));
```

Comment:
I think this chunk of code creating token fixtures is repeated enough times to make it worthwhile considering generalising it.

Reply:
i added a function here [a771b40](https://github.com/JanssenProject/jans/pull/9982/commits/a771b403ad2b86ad5eecccef508d4bae6bf4e9c4)